packages/shared/src/lib/user.ts

```diff
@@ -119,6 +120,13 @@ export type UserFlagsPublic = Partial<{
   showPlusGift: boolean;
 }>;
 
+export type UserSubscriptionFlags = Partial<{
+  provider: SubscriptionProvider;
```

Comment:
How is this decided? iirc this subscription flag is filled when the user subscribes. However, the user can be from iOS but then used webapp. Would it affect anything?

Reply:
It's set on API side. It has already been updated to set `paddle` for paddle subscriptions, and all have been updated retroactively.

Reply:
> However, the user can be from iOS but then used webapp. Would it affect anything?

It is used to determine what link the manage plus goes to. So if you subscribed on iOS and click "manage subscription" on web, it'll take you to a page that tells you to manage via AppStore.

Reply:
Amazing. Thanks for the clarification.